build.gradle

```diff
@@ -32,6 +32,15 @@ buildscript {
     classpath "com.pedjak.gradle.plugins:dockerized-test:0.5.6.35-SNAPSHOT"
     classpath 'com.github.ben-manes:gradle-versions-plugin:0.17.0'
     classpath "com.netflix.nebula:nebula-project-plugin:4.0.1"
+      
+    classpath 'io.netty:netty-handler-proxy:4.1.22.Final'
+    classpath 'io.netty:netty-codec-http:4.1.22.Final'
+    classpath 'io.netty:netty-handler:4.1.22.Final'
+    classpath 'io.netty:netty-transport-native-epoll:4.1.22.Final'
+    classpath 'io.netty:netty-transport-native-kqueue:4.1.22.Final'
+    classpath 'org.apache.commons:commons-lang3:3.3.2'
+    classpath 'org.apache.maven:maven-artifact:3.3.3'
+    classpath 'com.github.docker-java:docker-java:3.1.2-GEODE'
```

Comment:
Since the plugin is actually in buildSrc, I don't believe that buildSrc itself can export or declare its own dependencies. I could not see how to do that. The only thing it produces is `buildSrc.jar` which would need to be 'shaded' perhaps?

Reply:
I think I was wrong on this and we don't need them at all. I guess the `buildSrc` dependencies are actually carried over.